src/viadot/sources/cloud_for_customers.py

```diff
@@ -318,7 +318,7 @@ def to_df(
         tests: dict[str, Any] = kwargs.get("tests", {})
 
         url = url or self.url
-        records = self.extract_records(url=url)
+        records = self.extract_records(url=url, report_url=self.report_url)
```

Comment:
Why use different logic than for `url`? Would be simpler if both are treated the same way.

Reply:
@fdelgadodyvenia could you explain?

Reply:
Added issue : https://github.com/dyvenia/viadot/issues/1081